build-docs-snapshots

```diff
@@ -167,7 +167,9 @@ broken_lang ()
 		pt_BR)
 			toaddr="doc-pt-br@lists.php.net"
 		;;
```

Comment:
Can we drop the old one?

Reply:
That should in general be possible :grin: